backend/apps/nest/migrations/0002_alter_user_groups_alter_user_is_active_and_more.py

```
@@ -0,0 +1,29 @@
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+        ('nest', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='user',
+            name='groups',
+            field=models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.group', verbose_name='groups'),
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)